Artifacts/windows-powershell3/install.cmd

```diff
@@ -1,14 +1,45 @@
 @echo OFF
 
-reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set OSVERSION=32BIT || set OSVERSION=64BIT
+rem This is required to ensure we don't get error 0x8024800C -2145091572 WU_E_DS_LOCKTIMEOUTEXPIRED The data store section could not be locked within the allotted time.
+echo Cleaning software distribution folder.
+net stop "Windows Update" >nul 2>&1
+echo "Y" | del /f /s C:\Windows\SoftwareDistribution\ >nul 2>&1
+net start "Windows Update" >nul 2>&1
 
-if %OSVERSION%==32BIT (
```

Comment:
You could remove the "reg query" and simplify a bit with something like this:
if defined PROGRAMFILES(X86) (
REM do 64bit stuff
) else (
REM do 32bit stuff
)

Reply:
Yep. I'll ask the OP if there was a reason to run the query. Thanks.

Reply:
No real reason. So, I made the change. It seemed to work fine in Windows Server 2008 R2 SP1.